packages/utilities/components/interactionMode.ts

```diff
@@ -0,0 +1,38 @@
+/*
+    Mostly taken from the way Blueprint handles focus styles:
+    https://blueprintjs.com/docs/#core/accessibility.focus-management
+*/
+
+const TAB_KEY_CODE = 9;
+
+export class InteractionModeEngine {
+  constructor(
+    private container: Element,
+    private className: string,
+    private docEl: Element
+  ) {}
+
+  public start() {
+    this.container.classList.add(this.className);
+    this.docEl.addEventListener("mousedown", this.handleMouseDown);
+  }
+
+  private reset() {
+    this.container.classList.add(this.className);
+    this.docEl.removeEventListener("keydown", this.handleKeyDown);
+    this.docEl.removeEventListener("mousedown", this.handleMouseDown);
+  }
+
+  private handleKeyDown = e => {
+    if (e.which === TAB_KEY_CODE) {
+      this.reset();
+      this.docEl.addEventListener("mousedown", this.handleMouseDown);
+    }
+  };
```

Comment:
The linter in CI is complaining about this semicolon.
> ERROR: /jenkins/workspace/tend_UI-Kit_Pipeline_PR-255-LNJE7LCH5OIWLFWM5ZD2BNVWJSL3Q6C22KYJBBKQ6KMVE3NZUWSA/packages/utilities/components/interactionMode.ts[31, 4]: Unnecessary semicolon

Reply:
Yea, I know. I've gotta figure out a way to shut it up